Enhancement Probes (#1004)
ExtremeFLOW/neko: doc/pages/simcomps.md

 ## List of simulation components

 ## vorticity
 Computes the vorticity field an stores in the field registry as `omega_x`,
 `omega_y` and `omega_z`.
 Currently produces no output.

 ## lambda2

>  Computes Lambda2 for the velocity field and stores it in the normal output files as the first unused field.

Reviewer: ```suggestion
 Computes \f$ \lambda_2 \f$ for the velocity field and stores it in the normal output files as the first unused field.
```

>  This means that lambda2 can be found in the temeprature field in then fld files if running without a scalar

Reviewer: ```suggestion
 This means that \f$ \lambda_2 \f$ can be found in the temeprature field in then fld files if running without a scalar
```

 and s1 if neko is run with one scalar.
 
 ## probes

>  Probes the fluid field in the points given by an input file. Example usage:

Reviewer: ```suggestion
 Probes selected solution fields at the points given inside an input file. Example usage:
```
